internal/api/user_group_handler.go

```
@@ -655,15 +648,10 @@ func (h *UserGroupHandler) ValidateUserGroupQuery(ctx context.Context, req *conn
 		return nil, err
 	}
 
-	validationErr, err := h.store.Queries().ValidateUserGroupQuery(ctx, req.Msg.Query)
-	if err != nil {
-		return nil, apiErrorCtx(ctx, ErrInternal, connect.CodeInternal, "failed to validate query")
-	}
-
-	if validationErr != "" {
+	if err := dynamicquery.ValidateUserQuery(req.Msg.Query); err != nil {
 		return connect.NewResponse(&pm.ValidateUserGroupQueryResponse{
```

golangci-lint (2.12.2)

[error] error is not nil (line 651) but it returns nil

(nilerr)